docs/release-notes/10.1-beta.md

```diff
@@ -0,0 +1,391 @@
+  - GCC 14.3.1
+  - GCC Toolset 15
+  - LLVM Toolset 20.1.8
+  - Rust Toolset 1.88.0
+
+- Performance tools & debuggers:
+  - GDB 16.3
+  - Valgrind 3.25.1
+  - Bpftrace 0.23.5
+  - rsyslog 8.2506.0
+
+- Security updates:
+  - OpenSSL 3.5.1
+  - SELinux-policy 42.1.7
+  - SSSD 2.11.1
+  - Crypto-policies 20250804
+
+- Networking tools:
+  - NetworkManager 1.54.0
+  - iproute 6.14.0
+  - ethtool 6.15
+
+- Containers & virtualization:
+  - podman 5.6.0
+  - buildah 1.41.3
+  - Libvirt 11.5.0
+  - QEMU-KVM 10.0.0
+
+- Other updated components:
+  - Python 3.12.11
+  - Node.js 24
+  - Samba 4.22.4
+  - Mesa 25.0.7
+
+### Deviations from RHEL 10
+
+- Added Btrfs support, including ability to boot from Btrfs volume.
+- Added x86-64-v2 architecture.
+  - RHEL is increasing the architecture version baseline to `v3` dropping support for numerous older CPUs (and some newer ones). In AlmaLinux OS 10, we will follow Red Hat’s decision to ship x86-64-v3 optimized binaries by default, but we will also provide additional x86-64-v2 architecture **ONLY** for older hardware. All 3rd party packages for RHEL10 will target x86-64-v3, while the x86-64-v2 release of AlmaLinux OS 10 will only be suitable for workloads where using the default OS package set is enough, or where users will be able to rebuild any additional packages they require for x86-64-v2 architecture themselves.
+  - ALESCo is currently considering the option of rebuilding EPEL for x86-64-v2 users. If you use this version, please [let us know](https://github.com/almalinux/alesco), so we can make informed decisions about this version in the future.
+- Re-enabled frame pointers by default. This enables system-wide real-time tracing and profiling to optimize the performance of any workload running on AlmaLinux OS.
+- Re-enabled SPICE support for both server and client applications.
+- Enabled KVM for IBM POWER in the virtualization stack, so it’s possible to continue using KVM just like it is in AlmaLinux 8.
+- Firefox and Thunderbird in the system repositories are also shipped as regular RPM packages.
+- The following device drivers were modified to re-add PCI IDs for hardware that was previously disabled in our upstream:
+  - **aacraid** - Dell PERC2, 2/Si, 3/Si, 3/Di, Adaptec Advanced Raid Products, HP NetRAID-4M, IBM ServeRAID & ICP SCSI
+  - **be2iscsi** - Emulex OneConnectOpen-iSCSI for BladeEngine 2 and 3 adapters
+  - **be2net** - Emulex BladeEngine 2 and 3 adapters \_
+  - **hpsa** - HP Smart Array Controller
+  - **lpfc** - Emulex LightPulse Fibre Channel SCSI
+  - **megaraid_sas** - Broadcom MegaRAID SAS
+  - **mlx4_core** - Mellanox Gen2, ConnectX-2, ConnectX-3 adapters
+  - **mpt3sas** - LSI MPT Fusion SAS 3.0
+  - **mptsas** - Fusion MPT SAS Host
+  - **qla2xxx** - QLogic Fibre Channel HBA
+  - **qla4xxx** - QLogic iSCSI HBA
+    For a complete list of hardware support for which was added in this release, see the [Extended hardware support](#extended-hardware-support) section.
```

Comment:
I think this indentation is wrong, this line will get merged with the previous bullet point.